src/main/java/org/vamae/controller/GameBoardController.java

```diff
@@ -0,0 +1,29 @@
+package org.vamae.controller;
+
+import org.vamae.entity.GameBoard;
+import org.vamae.enums.Piece;
+
+import java.util.List;
+
```

Comment:
There should be an @AllArgsConstructor annotation

Reply:
Added @AllArgsConstructor annotation